pennylane/tape/qscript.py

```diff
@@ -168,12 +171,12 @@ def _unflatten(cls, data, metadata):
 
     def __init__(
         self,
-        ops=None,
-        measurements=None,
-        shots: Optional[Union[int, Sequence, Shots]] = None,
+        ops: Iterable[Operator] = (),
+        measurements: Optional[Iterable[MeasurementProcess]] = None,
```

Comment:
```suggestion
        measurements: Iterable[MeasurementProcess] = (),
```
Might as well make the change here too...